lib/Analysis/Utility.cpp

```diff
@@ -409,8 +437,7 @@ bool supportMFMA(triton::DotOp op) {
   if (!supportMFMAGranularity(aShape[0], bShape[1], aShape[1]))
```

Comment:
It seems to me that `supportMFMA()` should only check if matrix cores can be used to compute the dot, since it is only used by AccelerateAMDmatmul to see if we can use mfmalayout on C/D of dot. Checking types makes sense to me but checking granularity does not.

Reply:
Why?
If matrix is too small, we can not use mfma, even if data types are supported.